### ISSUE TITLE
[Remindme/Others] Embed field limits are not respected

### ISSUE THREAD

**Issue by Vexed01** [PhasecoreX/PCXCogs#46]
Embed field value limits are not respected. I set a long-ish reminder and tried to view it, however, I was unable to:

```
Exception in command 'reminder list'
Traceback (most recent call last):
  File "/home/pi/.pyenv/versions/vance/lib/python3.8/site-packages/discord/ext/commands/core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/pi/red-files/vance/cogs/CogManager/cogs/remindme/commands/reminder.py", line 110, in list
    await embed_splitter(embed, author)
  File "/home/pi/red-files/vance/cogs/CogManager/cogs/remindme/pcx_lib.py", line 62, in embed_splitter
    await destination.send(embed=embed)
  File "/home/pi/.pyenv/versions/vance/lib/python3.8/site-packages/discord/abc.py", line 1064, in send
    data = await state.http.send_message(channel.id, content, tts=tts, embed=embed,
  File "/home/pi/.pyenv/versions/vance/lib/python3.8/site-packages/discord/http.py", line 254, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In embed.fields.0.value: Must be 1024 or fewer in length.
In embed.fields.1.value: Must be 1024 or fewer in length.
```

I took a brief look at your embed splitter and it appears that this could also affect other cogs.

**Comment by Vexed01** [comment]
Update - the reminder was never sent and after a quick test, it looks like it silently fails.

**Comment by PhasecoreX** [comment]
Yeah, I must have accidentally removed the code that checked for the reminder text being too long. So I can add that back easily, I just need to handle the current reminders that are too big and can't send.

**Comment by PhasecoreX** [comment]
Okay fixed with a1ac1d1. When you update and reload the cog all the stick messages that need sending will be sent. Additionally, you can now use the `[p]reminder list`. Thanks for catching that!
